### DISCUSSION TITLE
Request: Restore UI links for viewing Stargazers

### DISCUSSION THREAD

**Discussion by ds5678** [community/community#8720] [Other Feature Feedback, Questions, & Ideas] [answered]
Up until a couple days ago, it was quite easy to view lists of people who:
* Starred a repository (ie https://github.com/github/feedback/stargazers)
* Forked a repository (ie https://github.com/github/feedback/network/members)
* Watch a repository (ie https://github.com/github/feedback/watchers)

As I've shown above, it's still possible to view these things, but the convenient UI button for each one was removed.

**Comment by bailey** [comment, marked as answer]
Hi there @ds5678! We just moved those links into the `About` section on the repo page as seen below. Hope this helps!

![image](https://user-images.githubusercontent.com/6895176/146112273-62a50c83-9a63-4d0c-ae95-260d10ace9ab.png)

**Reply by ds5678** [reply]
Oh, my bad. Thanks for letting me know

**Reply by upupming** [reply]
Feel like I do not know GitHub at all after the UI update, 😅

**Reply by thclark** [reply]
I love GitHub but the last rounds of UI updates have really felt like classic "moving stuff to show we've done something".

It doesn't take a UX expert to tell you that if you have a control component clearly showing star information, that's where the star information should be. Worse still, it's now a splitbutton, where the dropdown is something totally unrelated to stars, something to do with recommended lists??!!

It's not far down this path that GitHub becomes really hard to use, in a way that a year or two ago it was incredibly easy.

**Reply by phil294** [reply]
@bailey Could you please have another look at this? thclark raised some very valid points here, I think.

**Reply by brl0** [reply]
The color of those links causes me to overlook them when looking for links to the information in this request, which definitely gives these UI changes a bit of a hide a seek feel, which can be frustrating.  It might be helpful to make it more obvious that they are links.

**Reply by OlivierJM** [reply]
I had been looking for this and I just learned today from this post that this is a new way of getting stargazers, it is not a good user experience.

**Reply by nhooyr** [reply]
This is garbage UI/UX.

**Reply by mjmikulski** [reply]
OMG, if I have to google how to check stargazers or do other basic stuff, it means that UI is going in the wrong direction. What about having it in BOTH places? Leave it in the new place, so all managers, POs etc are happy, but allow also to view stargazers in the old way 😜

**Reply by thclark** [reply]
> OMG, if I have to google how to check stargazers or do other basic stuff, it means that UI is going in the wrong direction. What about having it in BOTH places? Leave it in the new place, so all managers, POs etc are happy, but allow also to view stargazers in the old way 😜

Yup. This!

**Reply by thclark** [reply]
@bailey and @phil294 was there ever any movement on this discussion from the GitHub side please?

**Reply by Monirhossen23** [reply]
Imo official assistant apply

**Comment by rungrotArunon** [comment]
Thanks you

**Reply by jonypeixoto** [reply]
Thank you very much!

**Comment by th3ken-dev** [comment]
thx

**Comment by jonypeixoto** [comment]
Thank you for the answer!

**Comment by Johnmogan** [comment]
#8720

**Comment by llvee** [comment]
Thank you all for reading, considering improvements. 

I am wondering if anyone here is interested in collaborating on extension engineering if nothing else in order to improve GH? 

I am mainly interested in engineering a social extension that can improve all the social features without having to update the main GH codebase. This will allow for community social features vs features that are only possible via internal GH presently which can help empower engineers more & contribute to more healthy engineering practices. I am also interested in collaborating on engineering an open source browser extension marketplace. 

The above would also help out the Github team, stakeholders by improving GH without needing to deploy a new public codebase while users are using the Github product. 

Does the above sound interesting to anyone? 

I noticed stargazers truly needs filters. Does anyone else see the value in adding filters to stargazers?
